Privacy Policy updates

rye opened the pull request:
This change is not entirely finished, but I have gotten started, and this is where I'm at right now.

Closes #2647.

rye committed:
Unflatten Markdown

There's no need to put everything on one line, and one of markdown's
key features it that is text and can hence be diffed nicely.

I like the one-sentence-per-line notion as a result of this.

rye committed:
privacy: Update introductory text

rye committed:
privacy: Add GDPR headings

rye committed:
Change "collecting" to "processing"

rye committed:
Fix privacy a bit more

hawkrives commented on data/privacy.md:
```diff
@@ -16,55 +79,71 @@ We may use the information we collect from you when you register, make a purchas
+We implement a variety of security measures when a user places an order enters, submits, or accesses their information to maintain the safety of your personal information.
+All transactions are processed through a gateway provider and are not stored or processed on our servers.
 
 ### Do we use 'cookies'?
```
we can probably just remove this section?

rye committed:
privacy: Remove section on "'cookies'"

rye committed:
privacy: Flatten text out into multiple lines

Signed-off-by: Kristofer Rye <kristofer.rye@gmail.com>

rye committed:
Merge branch 'master' into update-privacy

rye committed:
privacy: Delete everything

Signed-off-by: Kristofer Rye <kristofer.rye@gmail.com>

rye committed:
privacy: Take 2?

I like this style a lot more.  It seems like someone actually wrote it
and we didn't just take it from somewhere.

Signed-off-by: Kristofer Rye <kristofer.rye@gmail.com>

hawkrives reviewed (dismissed):
I like it!

Do we want to mention the "Submit WiFi problem" data collection as well?

drewvolz commented on data/privacy.md:
```diff
@@ -1,97 +1,73 @@
+unless as required by law**.  We treat your data as confidential and
+have the utmost respect for your privacy, and we think that sharing
+data with other companies for marketing purposes or compensation is a
+blatant violation of trust; we will never share these data.
```
> we will never share these data

Should this be “this data”? Data always confuses me as a singular or plural.

hawkrives replied:
If we leave it plural, can we make it "data points"?

drewvolz commented on data/privacy.md:
```diff
@@ -1,97 +1,73 @@
+  gather crash reports.  These tools are:
+  - _Google Analytics_, which can be disabled via the "Share
+    Analytics" toggle in the Settings view.
+  - _BugSnag_, which can also be disabled via the "Share Analytics"
```
I believe we still collect BugSnag reports from disabled analytics toggles... but I’d have to check the code.

drewvolz commented on data/privacy.md:
```diff
@@ -1,97 +1,73 @@
+  such a request, the following information is logged:
+  - _Your public IP address_, for diagnostic purposes.  We do not use
+    this to identify you or for tracking purposes, and this is logged
+    for diagnostic purposes.
```
Nit: “for diagnostic purposes” is repeated twice. Not the worst thing.

drewvolz commented on data/privacy.md:
```diff
@@ -1,97 +1,73 @@
+
+- We also use a few tools to track our most used features and to
+  gather crash reports.  These tools are:
+  - _Google Analytics_, which can be disabled via the "Share
```
Do we want to mention the detailed device information that’s being collected thru GA and BugSnag?

drewvolz replied:
@hawkrives raises a good point about the WiFi issue reporting. That’d be an important section to have.

drewvolz replied:
Oh, and something to think on. Do we want to say anything about OneSignal and notifications?

drewvolz replied:
1. Is it worth adding links to these tools?
2. Should we link to `ccc-server` so anyone can see its source?

rye replied:
I addressed most of your suggestions in this thread, but:

1. I don't think we really need to—this would just be distracting to me.
1. I think the link to `StoDevX/AAO-React-Native` is sufficient; anyone who wants to dig deeper will probably realize rather quickly how this works.  Since it's referred to as "our server" I don't think it'll really need much more detail.

Feel free to mark this conversation as unresolved if you think this is a blocker.

drewvolz reviewed (commented):
Thanks again. This is something I’ve kept coming back to because having it accurately reflect our practices is important to me. I have a few comments we can discuss.

rye committed:
privacy: Reword explanation for IP collection

Signed-off-by: Kristofer Rye <kristofer.rye@gmail.com>

rye committed:
privacy: these [data] -> this [data]

Signed-off-by: Kristofer Rye <kristofer.rye@gmail.com>

rye committed:
privacy: Address more tools and college systems

Signed-off-by: Kristofer Rye <kristofer.rye@gmail.com>

hawkrives reviewed (approved):
:shipit:

drewvolz reviewed (approved):
👍 :shipit:
